about.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/51f27b15da.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Abrehot Library</title>
+  </head>
+  <body>
+    <header class="font">
+      <nav class="star">
+        <i class="fa-sharp fa-solid fa-bars fa-xl hamburger"></i>
+      </nav>
+      <nav class="first-nav">
+        <ul>
+          <li>
+            <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </nav>
+      <nav class="second-nav">
+        <div>
+          <a href="index.html"><h2>Abrehot Library</h2></a>
+        </div>
+        <div>
+          <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Books</a></li>
+            <li><a href="#">Membership</a></li>
```

Comment:
- [ ] Dear @Yonasle, As mentioned by the previous code reviewer, your About page design is a bit different from the original design. In the original design, there is no background image. Also, you are not following the font type and sizing for the About page. As a front-end engineers, we are expected to deliver something identical. Please notice the difference and make the appropriate changes.
### Your design 
![image](https://user-images.githubusercontent.com/69077061/235098843-46957ad3-5b61-47a3-88b3-5c3291d51509.png)
### Original design 
![image](https://user-images.githubusercontent.com/69077061/235098957-9329e294-454a-4c3a-93d1-4445568aacd1.png)